consensus/poet/core/sawtooth_poet/poet_consensus/poet_block_verifier.py

```diff
@@ -123,6 +123,12 @@ def verify_block(self, block_wrapper):
                     utils.deserialize_wait_certificate(
                         block=block_wrapper,
                         poet_enclave_module=poet_enclave_module)
+                if wait_certificate is None:
+                    LOGGER.error(
```

Comment:
Good question.  I don't know if it serious enough to "take down" the validator (i.e., raise an exception) or if it is sufficient to return False to mean that this block should not be added to the chain.